ext/date/tests/bug81263.phpt

```diff
@@ -6,33 +6,51 @@ Bug #81263 (Wrong result from DateTimeImmutable::diff)
 $dt1 = new DateTimeImmutable('2020-07-19 18:30:00', new DateTimeZone('Europe/Berlin'));
 $dt2 = new DateTimeImmutable('2020-07-19 16:30:00', new DateTimeZone('UTC'));
 
-print_r($dt2->diff($dt1));
-print_r($dt1->diff($dt2));
+var_dump($dt2->diff($dt1));
+var_dump($dt1->diff($dt2));
```

Comment:
On the other hand, this one and the rest of the similar changes are not unintended: as the `from_string` property is `false` (which is converted to an empty string), there was a trailing whitespace in the `--EXPECT--` section which I wanted to avoid since it's a PITA to deal with. Are you fine with removing the trailing whitespaces?